src/utils/form-ui.ts

```diff
@@ -87,11 +87,28 @@ function getPredicateName(predicate: string): any {
   return null
 }
 
-async function getPropertyValue(field: string, property: string) {
+/**
+ * Fetch a value for a given subject and property, with special considerations for specific properties
+ * @param field
+ * @param property
+ * @param lang
+ */
+async function getPropertyValue(field: string, property: string, lang: string = 'en') {
   let propertyProxy: any
   const updatedProperty = changeHostProtocol(field)
 
-  if (property.includes('ui#values')) {
+  // For labels, loop over the values of the property. If there are multiple language values they will be returned here in the for await
+  // This is  the recommended way of fetching languages until a more permanent solution is implemented in ldflex
```

Comment:
Ideally, this links to the relevant issue in the LDflex repository, so future readers can check whether the permanent solution already exists.